core/utils/codel.h

```diff
@@ -135,7 +136,12 @@ class Codel final: public Queue<T> {
     return i;
   }
 
-  size_t Capacity() override { return queue_.max_size(); }
+  size_t Capacity() override { 
```

Comment:
I switched the underlying queue to deque which has a max size of the memory available to it according to the spec and I didn't see any constructor parameter to limit the deque size. In the use case of having many codels in drr handling a burst, it would be better to restrict queue size rather than have issues with memory although the max_entries can be specified as 0 to default to the memory size. I will add a comment but is allowing restriction not a good idea?